### PR TITLE
add docs section

### PR DESCRIPTION
**Pull request by ocefpaf**
Forgot this part in #234

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,16 +23,18 @@ matrix:
   include:
   - python: 2.7
     env: TEST_TARGET=default
-  - python: 3.6
+  - python: 3.7
```
Comment (ocefpaf) [inline .travis.yml]: Since this is not really testing anything let's bump it to be "future proof" once the test is available.

File: .travis.yml [hunk 2]
```diff
@@ -41,18 +43,17 @@ matrix:
...
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
-  - conda update conda
-  - conda config --set show_channel_urls true
-  - conda config --add channels conda-forge --force
-  - conda create -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
-  - source activate TEST
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      conda install configparser ;
+  - if [[ $TEST_TARGET != 'docs' ]]; then
```
Comment (ocefpaf) [inline .travis.yml]: We cannot build a python 3 env for `wofpy` yet b/c `odm2api` is not available, but `doctr` is a python 3 only library! So we need to have separate envs for testing and building the docs.

File: .travis.yml [hunk 3]
```diff
@@ -64,12 +65,12 @@ before_script:
...
 script:
   - if [[ $TEST_TARGET == 'default' ]]; then
-      py.test -s -rxs -v -k "not cbi" ;
+      pytest -s -rxs -v -k "not cbi" ;
     fi
 
   - if [[ $TEST_TARGET == 'build_cbi' ]]; then
       python wof/examples/flask/cbi/build_cbi_cache.py ;
-      py.test -s -rxs -v -k "cbi" ;
+      pytest -s -rxs -v -k "cbi" ;
```
Comment (ocefpaf) [inline .travis.yml]: Just syntax update, not a significant change.

File: .travis.yml [hunk 4]
```diff
@@ -87,7 +88,7 @@ script:
 
   - if [[ $TEST_TARGET == 'docs' ]]; then
       set -e ;
-      conda install doctr ;
+      conda install doctr sphinx;
```
Comment (ocefpaf) [inline .travis.yml]: The doc env is only `doctr` and `sphinx`.

**Comment by ocefpaf** [comment]
> I guess it looks ok ... Since I'm not in a position to know better.

This is the Travis-CI matrix item that we should watch for this PR:

https://travis-ci.org/ODM2/WOFpy/jobs/484509296

If that one passes we are OK.

**Comment by emiliom** [comment]
Thanks for the detailed explanations.

**Comment by ocefpaf** [comment]
Travis-CI is not starting :-/
Not sure what is going on, service outage or something wrong with the config file. I guess it is the former b/c we should see an error with the latter.

@emiliom while writing this I realize it is pointless to keep the coding standard and Python 3 boilerplate configurations. Are you OK removing them and keeping a simpler `.travis.yaml` file?

The rational is that fixing the coding standards is something that should be done with tools like `black` or no one will be willing to send the commits necessary to comply. Regarding Python 3 it is easy to enable when the code is ready for it.

**Comment by ocefpaf** [comment]
Oops, you were to fast! See my comment about Travis-CI not starting.

**Comment by emiliom** [comment]
> Oops, you were to fast! See my comment about Travis-CI not starting.

Sorry ...

> @emiliom while writing this I realize it is pointless to keep the coding standard and Python 3 boilerplate configurations. Are you OK removing them and keeping a simpler .travis.yaml file?
>
> The rational is that fixing the coding standards is something that should be done with tools like black or no one will be willing to send the commits necessary to comply. Regarding Python 3 it is easy to enable when the code is ready for it.

Sure. Sounds good.

**Pull request merged**